src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/openstack/OpenstackJCloudsProvider.java

```diff
@@ -195,6 +209,11 @@ private boolean isPublicKeyNameNotSet(Instance instance) {
                (instance.getCredentials().getPublicKeyName().isEmpty());
     }
 
+    private boolean useAutoGeneratedSecurityGroup(Instance instance) {
```

Comment:
So, NO SG should be automatically generated when no SG nor port to be opened are specified, right ?

Reply:
When none of SG and port are specified, each region will use its default security group, so yes, none sg should be auto generated in this case.